abacus_window.py

```diff
@@ -1305,6 +1305,8 @@ def value(self, count_beads=False):
 
     def label(self, string):
         ''' Label with the string. (Used with self.value) '''
+        if type(string) is not str:
+            string = str(string)
```

Comment:
I would simply keep this and remove every other individual `str()`, or the opposite. But is a personal preference.

Reply:
I will test it and then change the others soon. Thanks

Reply:
@tchx84 I have made the necessary changes. I had to fix another file called `sprites.py` and extend that string encoding change to it too,